packages/js/src/Modules/Verto/webrtc/Peer.ts

```diff
@@ -182,6 +182,23 @@ export default class Peer {
       this.iceGatheringComplete.resolve(true);
     }
   };
+  private handleConnectionStateChange = async (event: Event) => {
+    const { connectionState } = this.instance;
+    console.log(
+      `[${new Date().toISOString()}] Connection State`,
+      connectionState
+    );
+
```

Comment:
Not sure if this console.log is supposed to stay so i'm just leaving this comment as a doble check.